sdks/python/setup.py

```diff
@@ -121,6 +121,7 @@ def get_version():
     ('pyarrow>=0.11.1,<0.15.0; python_version >= "3.0" or '
      'platform_system != "Windows"'),
     'pydot>=1.2.0,<2',
+    'python-dateutil>=2.8.0,<2.9.0',
```

Comment:
Assuming that the package follows semantic versioning, <3 should be safe and increase future compatibility.
```suggestion
    'python-dateutil>=2.8.0,<3',
```